spyder/api/shellconnect/main_widget.py

```diff
@@ -1,14 +1,21 @@
-# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2021- Spyder Project Contributors
 #
-# Copyright © Spyder Project Contributors
-# Licensed under the terms of the MIT License
-# (see spyder/__init__.py for details)
+# Released under the terms of the MIT License
+# (see LICENSE.txt in the project root directory for details)
+# -----------------------------------------------------------------------------
 
 """
-Main widget to use in plugins that show content that comes from the IPython
-console, such as the Variable Explorer or Plots.
+Main widget for plugins showing content from the IPython Console.
+
+Used in the :guilabel:`Variable Explorer` and :guilabel:`Plots` plugins.
```

Comment:
```suggestion
Used in the :guilabel:`Variable Explorer`, :guilabel:`Plots` and other plugins.
```

The Debugger and Profiler are other examples.